app/helpers/application_helper.rb

```diff
@@ -1,2 +1,9 @@
 module ApplicationHelper
+  def full_title page_title = ""
+   <%= t.".base_title">
```

Comment:
trong file rb thì k cần nhúng <%= %>
```suggestion
   base_title = t ".base_title"
```